azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/ClientInfo.java

```diff
@@ -0,0 +1,35 @@
+package com.microsoft.azure.eventhubs;
+
+import com.microsoft.azure.eventhubs.amqp.AmqpConstants;
+import org.apache.qpid.proton.amqp.Symbol;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ClientInfo {
```

Comment:
add Javadoc - a guidance statement like - generic frameworks like yours or other famous examples - are the candidates who should use this.

Reply:
Is this fine?
/**
 * {@link ClientInfo} can be used to append the user's product information onto the metrics collected by the SDK
 *
 * Usage example:
 * AzureEventHub Logstash Input plugin (https://github.com/Azure/azure-diagnostics-tools/tree/master/Logstash/logstash-input-azureeventhub)
 */